CalendarApp/GoogleCalendarService/Manager/EventManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Google.Apis.Calendar.v3.Data;
+using GoogleCalendarService.Extensions;
+
+namespace GoogleCalendarService.Manager
+{
+    public class EventManager : IEventManager
+    {        
+        private readonly IBookingService _bookingService;
+
+        public EventManager(IBookingService bookingService)
+        {
+            _bookingService = bookingService;
+        }
+
+        public List<Event> CheckEvent(string calendarId, Event eventToAdd, bool forcedEvent)
+        {
+            if (forcedEvent)
+                return new List<Event>();
```

Comment:
Can we consider removing forcedEvent parameter and simply avoid calling this method if we are planning to do a force update?